spec/rubocop/cop/layout/comment_indentation_spec.rb

```diff
@@ -292,4 +292,55 @@ def compile_sequence(seq, seq_var)
       RUBY
     end
   end
+
+  context 'when `Layout/AccessModifierIndentation EnforcedStyle: outdent`' do
+    let(:allow_for_alignment) { true }
+    let(:indentation_width) { 2 }
```

Comment:
I would suggest removing this line (and from the top of the file) and inlining its value into the call, because it is never changed in the test file.